app/views/runs/show.html.erb

```diff
@@ -59,11 +53,16 @@
             </div>
             </div>
         <%- end %>
```

Comment:
I thought <div> was just for marking sections to apply css? Don't understand why this would need to be nested within a table.